app/assets/json/translations/locale-en.json

```diff
@@ -319,8 +311,6 @@
         "EMAIL_CONFIRMATION": "Email confirmation does not match",
         "EMAIL_NOT_FOUND": "Email not found. Please make sure your email has an account with us and is confirmed.",
         "FILE_MISSING": "Please upload a valid document",
-        "FILE_UPLOAD": "The file is too large or not a supported file type",
```

Comment:
This doesn't seem right `FILE_UPLOAD` and `FILE_UPLOAD_FAILED` are both being used at the FileUploadService no?